aa-sdk/core/src/actions/smartAccount/internal/waitForUserOperation.ts

```diff
@@ -0,0 +1,56 @@
+import type { Chain, Client, Hex, Transport } from "viem";
+import type {WaitForUserOperationTxParameters} from "../types";
+import {isBaseSmartAccountClient} from "../../../client/isSmartAccountClient";
+import {IncompatibleClientError} from "../../../errors/client";
+import {Logger} from "../../../logger";
+import {FailedToFindTransactionError} from "../../../errors/transaction";
```

Comment:
⚠️ **[eslint]** <[@typescript-eslint/no-unused-vars](https://typescript-eslint.io/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'FailedToFindTransactionError' is defined but never used.
<!-- __reviewdog__:ChAyNDYwMjAyODVmMTFkZmU5EgZyZGpzb24= -->